assets/js/adaptor-docs/hooks/useDocs.tsx

```diff
@@ -6,7 +6,7 @@ import { describePackage, PackageDescription } from '@openfn/describe-package';
 const cache: Record<string, PackageDescription | null | false> = {};
 
 const useDocs = (specifier: string) => {
-  const [docs, setDocs] = useState<PackageDescription | null | false>(null);
+  const [docs, setDocs] = useState<PackageDescription | false>();
```

Comment:
Ah, I need to revert this change

@elias-ba this is what was causing issues on my local build

`null` means something in particular with the docs, I think, and that pattern's been spread around a bit. I'll commit a small fix.